.github/workflows/release-with-changeset.yml

```diff
@@ -0,0 +1,141 @@
+# This action is centrally managed in https://github.com/asyncapi/.github/
```

Comment:
This is not centrally managed yet. Please remove the comment. 

```suggestion
```

Reply:
done.